database/db_tools.py

```diff
@@ -1,14 +1,14 @@
 import re
 import sys
 import csv
-import logging
 from os import listdir
 from os.path import join
 from collections import Counter
 from functools import lru_cache
 from typing import List, Set, Tuple, Dict, Callable, Any
 from tinydb import TinyDB, where
 from ratings import elo
+from utils.log import *
```

Comment:
I wouldn't recomending *, you could maybe do a `import utils.log as log` and use `log.info()`, so on. Could this be changed in the scrapers too?

Reply:
Yup, done